runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -559,7 +562,11 @@ public void onChatMessage(ChatMessage event)
 
 		}
 
-		if (config.showCannon() && (message.equals(CANNON_FURNACE_MESSAGE) || message.contains(CANNON_REPAIR_MESSAGE)))
+		if (config.showCannon() && (message.equals(CANNON_BASE_MESSAGE) ||
+			message.equals(CANNON_STAND_MESSAGE) ||
+			message.equals(CANNON_BARRELS_MESSAGE) ||
+			message.equals(CANNON_FURNACE_MESSAGE) ||
+			message.contains(CANNON_REPAIR_MESSAGE)))
```

Comment:
Our usual code style for continued lines is to push the operator to the next line instead of having it at the end of the line.
```suggestion
		if (config.showCannon() && (message.equals(CANNON_BASE_MESSAGE)
			|| message.equals(CANNON_STAND_MESSAGE)
			|| message.equals(CANNON_BARRELS_MESSAGE)
			|| message.equals(CANNON_FURNACE_MESSAGE)
			|| message.contains(CANNON_REPAIR_MESSAGE)))
```